openstack_lease_it/lease_it/backend/TestConnection.py

```diff
@@ -160,7 +174,10 @@ def _users(self):
 
         :return: dict()
         """
-        response = cache.get('users')
+        if GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
```

Comment:
Remplacer par
if not GLOBAL_CONFIG['RESET_CACHE_INSTANCES']:
response = cache.get('instances')